### PR TITLE
UPDATES: 1.4.1

### PR DESCRIPTION
**Pull request by sandyarmstrong**

### CODE REVIEW

**Comment by sandyarmstrong** [comment]
Added "Don't Merge" label because the link to ML workbooks is pointing to a branch.

File: UPDATES.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Version @BUILD_DOCUMENTATION_VERSION@
 
+* Add initial support for [ML.NET][ml-net]
+  ([#303](https://github.com/Microsoft/workbooks/issues/303)). See our new
+  [sample workbooks][ml-net-workbooks].
+* Add 64-bit .NET Core support on Windows (Mac was already 64-bit).
+* Bump Xamarin.Forms support to 3.0.0.482510.
+* Fix potential hangs loading iOS workbooks.
```
Comment (abock) [inline UPDATES.md]: Wasn't there an issue for this?

File: UPDATES.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # Version @BUILD_DOCUMENTATION_VERSION@
 
+* Add initial support for [ML.NET][ml-net]
+  ([#303](https://github.com/Microsoft/workbooks/issues/303)). See our new
+  [sample workbooks][ml-net-workbooks].
+* Add 64-bit .NET Core support on Windows (Mac was already 64-bit).
```
Comment (abock) [inline UPDATES.md]: Omit the Mac part. That's a given.

File: UPDATES.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+* Bump Xamarin.Forms support to 3.0.0.482510.
```
Comment (abock) [inline UPDATES.md]: Wasn't there an issue for this?

File: UPDATES.md (hunk 2 above)
Comment (sandyarmstrong) [reply UPDATES.md]: Fixed.

**Comment by sandyarmstrong** [comment]
Removed "don't merge" label now that the ML workbooks link is good. Added info about Mac support.

File: UPDATES.md [hunk 4]
```diff
@@ -1,5 +1,20 @@
...
+* Add initial support for [ML.NET][ml-net]
+  ([#303](https://github.com/Microsoft/workbooks/issues/303)).
+  - See our new [sample workbooks][ml-net-workbooks].
+  - Not yet supported on .NET Core on Mac.
+  - Console support on Mac requires a Mono release with
+    [this fix](https://github.com/mono/mono/issues/9033).
+* Add 64-bit .NET Core support on Windows.
+* Bump Xamarin.Forms support to 3.0.0.482510.
+* Fix potential hangs loading iOS workbooks.
+* Change Xcode location behavior to match VSmac
```
Comment (abock) [inline UPDATES.md]: VSmac → Visual Studio for Mac

File: UPDATES.md [hunk 5]
```diff
@@ -1,5 +1,20 @@
...
 
+* Add initial support for [ML.NET][ml-net]
+  ([#303](https://github.com/Microsoft/workbooks/issues/303)).
+  - See our new [sample workbooks][ml-net-workbooks].
+  - Not yet supported on .NET Core on Mac.
+  - Console support on Mac requires a Mono release with
+    [this fix](https://github.com/mono/mono/issues/9033).
+* Add 64-bit .NET Core support on Windows.
+* Bump Xamarin.Forms support to 3.0.0.482510.
+* Fix potential hangs loading iOS workbooks.
```
Comment (abock) [inline UPDATES.md]: * Fix a potential hang when loading an iOS workbook

**Pull request merged**